probe-dictionary/explorer.css

```diff
@@ -122,3 +122,15 @@ body {
 .hidden {
     display: none;
 }
+
+.contrast:after {
```

Comment:
I think this change should be in `new-pipeline/style/dashboards.css`, so we don't need to add a new CSS file to `index.html`.
We just need a class name here, i.e. `.contract {`. 
`.contrast` is a little bit generic though and it is not clear from the name what this is for. How about e.g. `portal-header-bar`?